src/components/search/Search.tsx

```diff
@@ -22,7 +49,7 @@ const Search = () => {
       params: {query: searchTerm, page: '1'},
```

Comment:
why is page still set to 1

Reply:
It is for initial data only. It is then getting updated in loadMoreMovies(){
    const newMovies = await fetchData(`/search/movie`, {
      params: {query: searchTerm, page: page + 1},
    });
}

Reply:
Please check Lines 19-39